sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -144,3 +144,47 @@
 .v-btn--variant-outlined {
   background-color: white !important;
 }
+
+// v-field label adjustment.Remove below style if its fixed in vuetify3.
+.v-field {
+  overflow: inherit !important;
+}
+
+// Active class for nav bar.
+.navigation-btn .active-btn {
+  color: $brand-blue !important;
+  background-color: $color-white !important;
+}
+
+// Btn toggle groups for nav bar adjustment.
+.navigation-btn.v-btn-group {
+  min-width: inherit;
+  overflow: inherit;
+}
+
+// Active sidebar class to match with Figma.
+.active-sidebar-item.v-list-item--active > .v-list-item__overlay {
+  opacity: 0 !important;
+}
+
+// Active sidebar class to match with Figma.
+.active-sidebar-item .v-list-item-title {
+  @extend .label-bold;
+}
+
+// Sidebar class to match with Figma.
+.sidebar-item .v-list-item-title {
+  @extend .nav-item-label;
+}
+
+//  256px is the default width of v-navigation-drawer,
+//  if its changes, adjust here too.
+// ref: https://next.vuetifyjs.com/en/api/v-navigation-drawer/#props-width
+.no-sidebar {
```

Comment:
This is a custom class right? why did we create this in vuetify.scss. I was under the impression that vuetify.scss has overwritten vuetify style classes to match figma.

Reply:
yup makes sense.moving